lib/DBDish/mysql/Native.pm6

```diff
@@ -2,7 +2,7 @@ use v6;
 
 unit module DBDish::mysql::Native;
 use NativeCall;
-use NativeLibs;
+use NativeLibs:auth<github:salortiz>;
```

Comment:
You probably don't need to qualify this, as long as there's a single copy in the ecosystem (which will happen when this is released)

Reply:
Yeah. I needed that to confirm zef could install DBIish and run tests before the release. It's been removed.